src/main/java/net/snowflake/client/core/SFBaseSession.java

```diff
@@ -682,6 +688,18 @@ public void setQueryContextCacheSize(int queryContextCacheSize) {
     this.queryContextCacheSize = queryContextCacheSize;
   }
 
+  public boolean getJdbcEnablePutGet() {
+    return jdbcEnablePutGet;
+  }
+
+  public void setJdbcEnablePutGet(boolean jdbcEnablePutGet) {
+    this.jdbcEnablePutGet = jdbcEnablePutGet;
+  }
+
+  public boolean getEnablePutGet() {
```

Comment:
do we need to have a setter for client side `enablePutGet` value?

Reply:
That was not the requirement but I can add it.

Reply:
Done